### PR TITLE
IB-47 created services to bike, user and booking

### PR DESCRIPTION
**Pull request by felipedreis**

### CODE REVIEW

File: backend/src/models/bike.model.ts [hunk 1]
```diff
@@ -5,11 +5,12 @@ enum BikeStatus {
```
Comment (garcialucasm) [inline backend/src/models/bike.model.ts:5]: please review optional properties on interfaces

File: backend/src/test/booking.service.test.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  beforeEach(async () => {
+    await bikeService.createBike(1, 'medium')
+  })
+
+  it('should create a student booking', async () => {
+    const userName = 'testuser'
+    const bikeSize = 'medium'
+    const room = 'A101'
+    const booking = await bookingService.createStudentBooking(userName, room, bikeSize)
```
Comment (garcialucasm) [inline backend/src/test/booking.service.test.ts]: please add remaining assert.strictEqual

File: backend/src/models/bike.model.ts (hunk 1 above)
Comment (felipedreis) [reply backend/src/models/bike.model.ts:5]: Did it. Thanks for pointing.

File: backend/src/test/booking.service.test.ts (hunk 2 above)
Comment (felipedreis) [reply backend/src/test/booking.service.test.ts]: Did it. Again, thanks.

**Review by garcialucasm — APPROVED** [review]

**Pull request merged**